doc/experimental/phase1/phase1.md

```diff
@@ -238,6 +238,23 @@ kubectl patch vz -n $VZCR_NS $VZCR -p '{"metadata":{"finalizers":[]}}' --type=me
 kubectl delete vz -n $VZCR_NS $VZCR
 ```
 
+## Delete the VerrazzanoMonitoringInstance
+This section describes how to delete the VerrazzanoMonitoringInstance, which is no longer needed.
```

Comment:
The draft number of the doc should be updated too

Reply:
fixed